billboardd/daemon.h

```diff
@@ -295,6 +303,13 @@ class Daemon : public QObject {
             context->subscribeTo(interestingProperties);
             gconfmon->subscribeTo(interestingProperties);
 
+            if (interestingProperties.contains("tklock")) {
+                QObject::connect(&tklock, SIGNAL(locked()),
```

Comment:
This doesn't update when the `unlocked()` signal is emitted - I'll comment below how to change this, should be `SIGNAL(changed())` here, so we don't need two signals.